src/sentry/static/sentry/app/views/organizationDiscover/result/index.jsx

```diff
@@ -77,6 +77,10 @@ class Result extends React.Component {
     }
   }
 
+  shouldComponentUpdate(nextProps, nextState) {
+    return !isEqual(nextProps, this.props) || !isEqual(nextState, this.state);
```

Comment:
Are there any performance implications?

Reply:
Not afaik, I don't think props and state are particularly huge objects, it's reducing quite a bit of unnecessary rendering though